articles/mdm-migration.md

```diff
@@ -6,185 +6,108 @@ This guide provides instructions for migrating devices from your current MDM sol
 
-### Step 2: choose migration workflow and migrate hosts
+### Step 3: choose migration workflow and migrate hosts
 
 There are two migration workflows in Fleet: default and end user.
 
 The default migration workflow requires that the IT admin unenrolls hosts from the old MDM solution before the end user can complete migration. This will result in a gap in MDM coverage until the end user completes migration.
 
-The end user migration workflow allows the end user to kick-off migration by unenrolling from the old MDM solution on their own. Once the user is unenrolled, they're prompted to turn on MDM features in Fleet. This reduces the gap in MDM coverage.
-
-Configuring the end user migration workflow requires a few additional steps.
+The end user migration workflow allows the end user to kick off migration by unenrolling from the old MDM solution on their own. Once the user is unenrolled, they're prompted to turn on MDM features in Fleet. This reduces the gap in MDM coverage.
 
 #### Default workflow
 
-1. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
+End user experience:
 
-2. Send [these guided instructions](#how-to-turn-on-mdm-default) to your end users to complete the final few steps via Fleet Desktop.
-    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old MDM and when the host turns on MDM in Fleet.
+Once a host is unenrolled from the old MDM solution, the end user will be given the option to download the MDM enrollment profile on their **My device page**.
 
-##### End user experience
+![Fleet icon in menu bar](https://raw.githubusercontent.com/fleetdm/fleet/main/website/assets/images/articles/fleet-desktop-says-hello-world-cover-1600x900@2x.jpg)
 
-1. The end user will receive a "Device Enrollment: &lt;organization&gt; can automatically configure your Mac." system notification within the macOS Notifications Center. 
-   
-2. After the end user clicks on the system notification, macOS will open the **System Setting > Profiles** and ask the user to "Allow Device Enrollment: &lt;organization&gt; can automatically configure your Mac based on settings provided by your System Administrator."
-  
-3. If the end user does not install the profile, the system notification will continue to prompt the end user until the setting has been allowed.
-   
-4. Once this setting has been approved, the MDM enrollment profile cannot be removed by the end user.
+<img width="1400" alt="My device page - turn on MDM" src="https://user-images.githubusercontent.com/5359586/229950406-98343bf7-9653-4117-a8f5-c03359ba0d86.png">
 
-##### How to turn on MDM (default)
+If the host is in ABM, the end user will be prompted w/ Apple's **Remote Management** full-screen popup.
```

Comment:
Hey @jahzielv is this how it works today? If a host is in ABM, assigned to Fleet in ABM, enrolled to Fleet, and unenerolled from the old MDM, Fleet pops up Apple's Remote Management dialog?

If so, I think we want to add a screenshot here so IT admins can see what that looks like.

Reply:
Hrm, good call out. I don't think the Remote Management dialog is popped up unless the host goes through what we're calling the "end-user workflow". The dialog only comes up if `profiles renew -type enrollment` is run (which we do via fleetd in the end-user workflow) I think we should remove this sentence.

Reply:
@noahtalerman nvm!! I just went through this flow and found that I was wrong.

The "Remote Management" dialog does in fact pop up after refetching the host. I forgot that Fleet checks (independently of the migration flow) if a host needs to run `profiles renew` and prompts fleetd to do so if needed.

I can add a screenshot.

Reply:
Thank!